app/cli/cmd/root.go

```diff
@@ -83,6 +93,38 @@ func NewRootCmd(l zerolog.Logger) *cobra.Command {
 			return nil
 		},
 		PersistentPostRunE: func(cmd *cobra.Command, args []string) error {
+			// Do not track telemetry if the env variable is set
+			var telemetryClient telemetry.Client
+			if os.Getenv(doNotTrackEnv) == "1" || os.Getenv(doNotTrackEnv) == "true" {
+				logger.Debug().Msg("Telemetry is disabled")
```

Comment:
we might not need this debug statement but instead put one when it's enabled so they know how to disable it. It will also remove the if/else